frontend/src/metabase/query_builder/components/filters/modals/InlineOperatorSelector/InlineOperatorSelector.styled.ts

```diff
@@ -1,13 +1,28 @@
 import styled from "@emotion/styled";
-import { space } from "metabase/styled-components/theme";
+import {
+  space,
+  breakpointMinHeightMedium,
+} from "metabase/styled-components/theme";
 import { color, alpha, darken, lighten } from "metabase/lib/colors";
 
 export const InlineOperatorContainer = styled.div`
   font-weight: bold;
-  font-size: 1rem;
+  font-size: 0.875rem;
+  ${breakpointMinHeightMedium} {
+    font-size: 1rem;
+  }
   margin-bottom: 0.875rem;
   display: inline-flex;
   align-items: flex-start;
+
+  .field-icon {
```

Comment:
To avoid using global CSS name like this, please wrap `Icon` with a styled-component and use it here, like this:
```
import { FieldIcon } from "../InlineOperatorSelector/InlineOperatorSelector.styled"

... 

${FieldIcon} {
  width: 16px;
  ...
}
```

in `InlineOperatorSelector.styled.tsx`:

```
export const FieldIcon = styled(Icon)`
  margin-right: 0.5rem;
`;
```

Reply:
@iethree fyi